libs/sdk-ui-dashboard/src/model/commandHandlers/drill/common/filterValuesResolver.ts

```diff
@@ -150,7 +150,8 @@ async function resolveAttributeFilterValues(
         elementsPage = await elementsPage.next();
     }
     return elements.reduce((map, element) => {
-        map[element.uri] = element.title;
+        const key = element.uri === null ? "null" : element.uri; //TODO not sure how to handle null key
+        map[key] = element.title;
```

Comment:
This is unnecessary, you can index by null no problem.

Reply:
Typescript complains about it ... I have checked how internally indexing working and thats why I changed to string ... but maybe I should use String for it

Reply:
fixed key converted in case Null to string "null" comment added